cdk/lib/recipes-backend.ts

```diff
@@ -83,15 +74,16 @@ export class RecipesBackend extends GuStack {
 			default: `/${this.stage}/${this.stack}/${app}/fastly-key`,
 		});
 
-		const telemetryXAR = new GuParameter(this, 'TelemetryCrossAcctRole', {
+		const telemetryTopic = new GuParameter(this, 'TelemetryTopic', {
 			fromSSM: true,
-			default: `/${this.stage}/${this.stack}/${app}/telemetryXAR`,
-			description: 'Cross-account role to allow data submissions',
+			default: `/${this.stage}/feast/recipe-structuriser/telemetryTopic`,
+			description:
+				'ARN of the SNS topic to use for data submissions (shared with structuriser)',
 		});
-		const telemetryTopic = new GuParameter(this, 'TelemetryTopic', {
+
+		const eventBusParam = new GuParameter(this, 'EventBus', {
 			fromSSM: true,
-			default: `/${this.stage}/${this.stack}/${app}/telemetryTopic`,
-			description: 'ARN of the SNS topic to use for data submissions',
+			default: `/${this.stage}/feast/feast-shared-infra/crier-event-bus`,
```

Comment:
For this and lines above: any reason `this.stack` wouldn't be `feast` in this context?